app/frontend/components/shared/permit-applications/requirement-form.tsx

```diff
@@ -97,7 +103,7 @@ export const RequirementForm = observer(
     const infoBoxData = permitApplication.diffToInfoBoxData
 
     useEffect(() => {
-      if (!usingCurrentTemplateVersion && currentUser.shouldSeeApplicationDiff) {
+      if (!usingCurrentTemplateVersion && userShouldSeeApplicationDiff) {
```

Comment:
This is a bit tricky as this has been changed in release 0.4.0

See:
https://github.com/bcgov/HOUS-permit-portal/blob/ee8a6ef63fff42250d4d6cc7ccceb08583b85627/app/frontend/components/shared/permit-applications/requirement-form.tsx#L112

and

https://github.com/bcgov/HOUS-permit-portal/blob/ee8a6ef63fff42250d4d6cc7ccceb08583b85627/app/frontend/models/permit-application.ts#L246

we likely just need to && this in once we get to solving conflicts

Reply:
yeah should be fine, thanks